internal/telemetry/tracker.go

```diff
@@ -34,7 +34,8 @@ type Tracker struct {
 	SystemNamespace string
 }
 
-const interval = 10 * time.Minute
+// Daily Heartbeat
+const interval = 1440 * time.Minute
```

Comment:
Use `24 * time.Hour`

Reply:
@a13x5 -- Thanks for the guidance.

@Schnitzel requested adding a default token so that if another party cloned the repo and built the binary that we'd [potentially] receive telemetry. If I understand your suggestion to use GH secrets, image builds done outside of at GH workflow would not be able to emit telemetry.